lms/templates/dashboard.html

```diff
@@ -78,10 +78,8 @@
 <main id="main" aria-label="Content" tabindex="-1">
     <div class="container dashboard" id="dashboard-main">
       <div class="my-courses" id="my-courses">
-        <header class="wrapper-header-courses">
-          <h2 class="header-courses">${_("My Courses")}</h2>
-        </header>
-
+        <%include file="learner_dashboard/_dashboard_navigation_courses.html"/>
+        <h2 class="hd hd-2 sr" id="courses-tab">${_('My Courses')}</h2>
```

Comment:
yeah you're right. I had this code from yesterday (but not pushed), before splitting the two files.